azure-pipelines.yml

```diff
@@ -0,0 +1,76 @@
+# Enable builds on all branches
+trigger:
+  branches:
+    include:
+      - master
+
+# Enable PR triggers that target the master branch
+pr:
+  autoCancel: true # cancel previous builds on push
+  branches:
+    include:
+      - master
+
+jobs:
+  - job: build_and_test
+    timeoutInMinutes: 120
+    pool:
+      vmImage: "Ubuntu-16.04"
+    steps:
+      - checkout: self
+
+      # Install DAML SDK, possibly from cache
+      - bash: |
+          set -euo pipefail
+          cat daml.yaml | grep -oP '(?<=sdk-version: ).*' > /tmp/SDK_VERSION
+      - task: CacheBeta@1
+        inputs:
+          key: /tmp/SDK_VERSION
+          path: /tmp/daml
+          cacheHitVar: DAML_CACHE_HIT
+      - bash: |
+          set -euo pipefail
+          DIR=$(pwd)
+
+          # Install DAML SDK
+          SDK_VERSION=$(cat /tmp/SDK_VERSION)
+          curl https://get.daml.com | sh -s $SDK_VERSION
+
+          mkdir -p /tmp/daml
+          cd $HOME/.daml
+          tar czf /tmp/daml/sdk.tar.gz *
+        condition: ne(variables.DAML_CACHE_HIT, 'true')
+        displayName: daml-cache-miss
+      - bash: |
+          set -euo pipefail
+          mkdir -p $HOME/.daml
+          cd $HOME/.daml
+          tar xzf /tmp/daml/sdk.tar.gz
+        condition: eq(variables.DAML_CACHE_HIT, 'true')
+        displayName: daml-cache-hit
+
+      # Build and test TypeScript
+      - bash: |
+          set -euo pipefail
+
+          yarn --version
+          export PATH=$HOME/.daml/bin:$PATH
+          daml build
+          daml codegen ts -o daml2ts -p package.json .daml/dist/*.dar
+          yarn install
+          yarn workspaces run build
+          yarn workspaces run lint
+          #CI=yes yarn workspaces run test --ci --all
```

Comment:
There doesn't seem to be a `test` task defined, so this won't run. I've left it here as a reminder, but feel free to delete if you don't intend to add tests.